warehouse/models/mart/gtfs_schedule_latest/dim_stops_latest.sql

```diff
@@ -4,6 +4,49 @@ dim_stops_latest AS (
     table_name = ref('dim_stops'),
     clean_table_name = 'dim_stops'
     ) }}
+),
+
+stg_state_geoportal__state_highway_network_stops AS (
+    SELECT *
+    FROM {{ ref('stg_state_geoportal__state_highway_network_stops') }}
+),
+
+
+buffer_geometry_table AS (
+    SELECT
+        -- equal to 100ft, as requested by Uriel
+        ST_BUFFER(wkt_coordinates,
+            30.48) AS buffer_geometry
+    FROM stg_state_geoportal__state_highway_network_stops
+),
+
+current_stops AS (
+    SELECT
+        pt_geom,
+        stop_id
+    FROM dim_stops_latest
+),
+
+
+stops_on_shn AS (
+    SELECT
+        current_stops.*
+    FROM buffer_geometry_table, current_stops
+    WHERE ST_DWITHIN(
+            buffer_geometry_table.buffer_geometry,current_stops.pt_geom, 0)
+),
+
+dim_stops_latest_with_shn_boolean AS (
+
+SELECT
+    dim_stops_latest.*,
+    IF(stops_on_shn.stop_id IS NOT NULL, TRUE, FALSE) AS exists_in_dim_stops_latest
+FROM
+    dim_stops_latest
+LEFT JOIN
+    stops_on_shn
+ON
+    dim_stops_latest.stop_id = stops_on_shn.stop_id
```

Comment:
Is joining on `stop_id` going to potentially cause some conflicts here? There may be two feeds that have the same stop_id that this could match on. Would a join on [`dim_stops_latest._gtfs_key`](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.dim_stops_latest#columns) be more appropriate?

Reply:
Hey @evansiroky – you're completely right. I must have left this in here after some comparative testing, apologies! I just made the swap and am re-requesting your review